src/astro/sql/parsers/sql_directory_parser.py

```diff
@@ -86,6 +87,8 @@ def render_single_path(
             operator_kwargs = set_kwargs_with_defaults(
                 front_matter_opts, conn_id, database, role, schema, warehouse
             )
+            if kwargs.get("params"):
+                operator_kwargs["params"] = kwargs.get("params")
```

Comment:
How would a user know that they can pass `params` dictionary to this function?

Reply:
@kaxil I made it an explicit parameter with docstring.